### PR TITLE
75 add message filter

### LINKED ISSUE #75: Add message-filter
**Meddelanden kan visas baserat på vem som har publicerat dem.**

**Additional context**
Wee need a way  ( maby in /message or /welcome ) so that we can filter the shown messages based on who wrote them.

### PR DESCRIPTION
**Pull request by Frlundsten**
### Related Issue
Closes #75

### Proposed Changes
- Add message filter to filter users.

### Description
- Added searchbutton to /messages.
- Added new endpoint /user and mesagesfromuser.html.
- Added function to script.js to show dropdown content.
- Updated messages.html
- Can go back by searching for all users

### CODE REVIEW

File: src/main/resources/static/search.jpg
Comment (Win-ther) [inline src/main/resources/static/search.jpg:1]: Lmao nice 😂

**Review by Win-ther — COMMENTED** [review]
Amazing work, I love that you used the ugly search thing I did in paint 😂
Maybe we want some way to go back to /messages from /user? Right now you can only get back by going to the profile and then going to messages again.

File: src/main/resources/static/search.jpg
Comment (Frlundsten) [reply src/main/resources/static/search.jpg:1]: Thx 😄 
You can go back by go back in browser but that might be cheating, will look in to that!
Also found a bug... Cannot change language after searching a user..

**Review by Win-ther — APPROVED** [review]
Good job, works really good ^^

**Review by altmannk — APPROVED** [review]
I LOVE THE SEARCH BOTTON 🥇  Great job! 😃

**Review by marcusju23 — APPROVED** [review]
Beautiful! 😄

**Pull request merged**